src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override public void doWork() {
+        System.out.println("Excavator started its work");
+    }
```

Comment:
method annotations should be above its declaration. Fix this and other annotation comment everywhere
Your style of annotations can precede some method **parameters** in the same line, but not the method